src/app/containers/MostRead/Canonical/index.jsx

```diff
@@ -1,39 +1,68 @@
-import React, { useEffect, useState } from 'react';
+import React, { useEffect, useState, useContext } from 'react';
 import 'isomorphic-fetch';
 import { string } from 'prop-types';
+import { MostRead } from '@bbc/psammead-most-read';
+import { ServiceContext } from '#contexts/ServiceContext';
 import webLogger from '#lib/logger.web';
+import { mostReadRecordIsFresh } from '../utilities';
+import LastUpdated from './lastUpdated';
 
 const logger = webLogger();
 
 const CanonicalMostRead = ({ endpoint }) => {
-  const [promos, setPromo] = useState([]);
-  const [data, setData] = useState({});
-
-  const handleResponse = async response => {
-    const mostReadData = await response.json();
-    setPromo(mostReadData.records.slice(0, 10));
-    setData(mostReadData);
-  };
+  const [items, setItems] = useState([]);
+  const {
+    service,
+    script,
+    dir,
+    datetimeLocale,
+    mostRead: { header, lastUpdated, numberOfItems },
+  } = useContext(ServiceContext);
 
   useEffect(() => {
+    const handleResponse = async response => {
+      const mostReadData = await response.json();
+      // do not show most read if lastRecordUpdated is greater than 35min as this means PopAPI has failed twice
+      // in succession. This suggests ATI may be having issues, hence risk of stale data.
+      if (mostReadRecordIsFresh(mostReadData.lastRecordTimeStamp)) {
+        const mostReadItems = mostReadData.records
+          .slice(0, numberOfItems)
+          .map(({ id, promo: { headlines, locators, timestamp } }) => ({
+            id,
+            title: headlines.shortHeadline,
```

Comment:
@OlgaLyubin pointed out that in the Ares data we have both `headline` and `shortHeadline`, yet only `shortHeadline` is used. This is a limitation of Ares with MostRead, we can't manipulate the data to get rid of this so this is just something that's a little out of our control.

Reply:
Where's the GraphQL version of Ares? :)

Reply:
Learning day? :D